### ISSUE TITLE
remote http domain name restriction

### ISSUE THREAD

**Issue by trungpham** [h2non/imaginary#62]
Can we restrict where to pull the remote image from? This way other people cannot use our server for their benefits?

**Comment by h2non** [comment]
Not yet. This is a feture request. It can easily supported.

**Comment by h2non** [comment]
Features shipped in `imaginary@0.1.22`. Feel free to update the package or pull the latest Docker image.

Usage is really simple:
```bash
$ imaginary -enable-url-source -allowed-origins "http://localhost,http://server.com,http://server2.com"
```

In case that the origin is not allowed the server will reply with a `400` status error.
For more info, see the [usage docs](https://github.com/h2non/imaginary#usage).
